yt/yt/server/queue_agent/consumer_controller.cpp

```diff
@@ -222,6 +222,15 @@ class TConsumerSnapshotBuildSession final
 
                 subConsumerPartitionSnapshot->NextRowIndex = consumerPartitionInfo.NextRowIndex;
                 subConsumerPartitionSnapshot->LastConsumeTime = consumerPartitionInfo.LastConsumeTime;
+
+                if (consumerPartitionInfo.ConsumerMeta) {
+                    if (auto cumulativeDataWeight = consumerPartitionInfo.ConsumerMeta->CumulativeDataWeight) {
```

Comment:
It is not necessary to check for null, since this should be our first time setting this value.